Escape Academy/escapeacademy.asl

```diff
@@ -2,63 +2,40 @@ state("Escape Academy") { }
 
 startup
 {
-	var bytes = File.ReadAllBytes(@"Components\LiveSplit.ASLHelper.bin");
-	var type = Assembly.Load(bytes).GetType("ASLHelper.Unity");
-	vars.Helper = Activator.CreateInstance(type, timer, this);
+	Assembly.Load(File.ReadAllBytes("Components/asl-help")).CreateInstance("Unity");
 
 	vars.Helper.AlertLoadless();

```

Comment:
Getting this error here:
```
LiveSplit.exe Error: 0 : 
Exception thrown: 'System.Reflection.TargetInvocationException' in 'startup' method:
Exception has been thrown by the target of an invocation.

   at ASL line 7 in 'startup'

   at LiveSplit.ASL.ASLMethod.Call(LiveSplitState timer, ExpandoObject vars, String& version, Double& refreshRate, Object settings, ExpandoObject old, ExpandoObject current, Process game) 
   at LiveSplit.ASL.ASLScript.RunStartup(LiveSplitState state) 
   at LiveSplit.UI.Components.ASLComponent.LoadScript() 
```
Occurs on line 9 if this is commented out.

Reply:
Not sure if this is a me problem or not, but I don't have the time this morning to diagnose it :P